public/400.html

```diff
@@ -81,21 +69,20 @@ <h1 class="heading-xlarge">This page cannot be found</h1>
 
         <div id="global-app-error" class="app-error hidden"></div>
 
-        <!-- <script src="/assets/govuk-template.js"></script> -->
-
-        <script src="/assets/application-521e3385d0bfe58be0f9ba973447d159de0c6df258a2d7b16991836f16c5006e.js"></script><script>(function (i, s, o, g, r, a, m) {
-                                                                                                                            i['GoogleAnalyticsObject'] = r;
-                                                                                                                            i[r] = i[r] || function () {
-                                                                                                                                (i[r].q = i[r].q || []).push(arguments)
-                                                                                                                            }, i[r].l = 1 * new Date();
-                                                                                                                            a = s.createElement(o), m = s.getElementsByTagName(o)[0];
-                                                                                                                            a.async = 1;
-                                                                                                                            a.src = g;
-                                                                                                                            m.parentNode.insertBefore(a, m)
-                                                                                                                        })
-                                                                                                                        (window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
-                                                                                                                        ga('create', 'UA-37377084-57', 'auto');
-                                                                                                                        ga('send', 'pageview');</script>
+        <script>
+            (function (i, s, o, g, r, a, m) {
```

Comment:
This function is really difficult to understand, are their more descriptive names we could give these parameters? 

Perhaps extract the anon function to a named function as well as it would be clearer to read.

Reply:
This is the standard GA code so has to be applied as is so they say - It has a complicated history. I won't pretend to understand it haha. Happy to explore options of putting it somewhere and tidying it but that would have to be a site-wide change. Not sure if it should block this little piece though.